command/flags.go

```diff
@@ -80,9 +80,11 @@ var daemonFlags = []cli.Flag{
 		Required: false,
 	},
 	&cli.BoolFlag{
-		Name:     "watch-config",
-		Usage:    "Watch for changes to config file and automatically reload",
-		EnvVars:  []string{"INDEXER_WATCH_CONFIG"},
+		Name:  "watch-config",
+		Usage: "Watch for changes to config file and automatically reload",
+		// Note: `INDEXER_WATCH_CONFIG` is added for backward compatibility.
+		//       Remove it ince `STORETHEINDEX_WATCH_CONFIG` is rolled out in both environments.
+		EnvVars:  []string{"INDEXER_WATCH_CONFIG", "STORETHEINDEX_WATCH_CONFIG"},
```

Comment:
@gammazero See commit [message](https://github.com/filecoin-project/storetheindex/pull/551/commits/96b284ed40a2d95d669c47c0490151cca8b2c818) on why we need to keep env var name changes backward compatible.